js/gratipay.js

```diff
@@ -11,12 +11,24 @@ Gratipay.getCookie = function(key) {
 }
 
 Gratipay.init = function() {
+    Gratipay.warnOffUsersFromDeveloperConsole();
     Gratipay.forms.initCSRF();
     Gratipay.signIn();
     Gratipay.signOut();
     Gratipay.payments.initSupportGratipay();
 };
 
+Gratipay.warnOffUsersFromDeveloperConsole = function()
+{
```

Comment:
For the sake of consistency, we should place this `{` on the same line as `function()`